packages/base-controller/src/ControllerMessenger.ts

```diff
@@ -154,7 +173,7 @@ export class RestrictedControllerMessenger<
    * @throws Will throw when a handler has been registered for this action type already.
    * @template T - A type union of Action type strings that are namespaced by N.
```

Comment:
This comment seems out of date now.

Reply:
Fixed here f1cc1236578ebc263cb14b44a04c4f6027938801

Reply:
Good catch! Fixed here: https://github.com/MetaMask/core/pull/1890/commits/f1cc1236578ebc263cb14b44a04c4f6027938801